prometheus_api_client/exceptions.py

```diff
@@ -5,3 +5,9 @@ class PrometheusApiClientException(Exception):
     """API client exception, raises when response status code != 200."""
 
     pass
+
+
+class MetricValueConversionFailed(Exception):
```

Comment:
Would you mind renaming this to something like `MetricValueConversionError` so that `Error` is a suffix and pep8 N818 error code is not thrown by precommit check?

Reply:
Updated :)